kubernetes/helm_charts/core/apimanager/values.j2

```diff
@@ -25,10 +27,22 @@ strategy:
   maxsurge: {{ apimanager_maxsurge|default('25%') }}
   maxunavailable: {{ apimanager_maxunavailable|default('25%') }}
 
+enable_apiproxyvalidator: {{ enable_apiproxyvalidator }}
+proxynetwork:
+  port: 9090
+  targetport: 9090
+proxyvalidatorresources:
+  requests:
+    cpu: {{proxyvalidator_cpu_req|default('100m')}}
+    memory: {{proxyvalidator_mem_req|default('100Mi')}}
+  limits:
+    cpu: {{proxyvalidator_cpu_limit|default('1000m')}}
+    memory: {{proxyvalidator_memory_limit|default('1024Mi')}}
+
 {{ apimanager_liveness_readiness | to_nice_yaml }}
 
 autoscaling:
-  enabled: {{ apimanager_autoscaling_enabled | default('false') }}
+  enabled: {{ apimanager_autoscaling_enabled | default('true') }}
```

Comment:
@SMYALTAMASH we are disabling the autoscaling for all other services by default. Any reason why you want to enable it for api-manger?

Reply:
disabled it @gandham-santhosh